### ISSUE TITLE
Specific remote queries not firing.

### ISSUE THREAD

**Issue by mdobson** [zettajs/zetta#222]
I can't get this query to work. I think this should still work?

```
//two servers Z1 and Z1 each connected to Z3.
//This app lives on Z3
module.exports = function(server) {
  var q1 = server.from('Z1').where({type: 'led'});
  var q2 = server.from('Z2').where({type: 'buzzer'});
  server.observe([q1, q2], function(led, buzzer) {
    console.log(arguments);
  });
}
```

This works though.

```
//two servers Z1 and Z1 each connected to Z3.
//This app lives on Z3
module.exports = function(server) {
  var q1 = server.from('*').where({type: 'led'});
  var q2 = server.from('*').where({type: 'buzzer'});
  server.observe([q1, q2], function(led, buzzer) {
    console.log(arguments);
  });
}
```

I've got a potential fix lined up if case number 1 is desired behavior.

**Comment by AdamMagaluk** [comment]
I'm seeing that as well, I would consider it an issue.

**Comment by mdobson** [comment]
Fixed with #223
